### ISSUE TITLE
Licenses

### ISSUE THREAD

**Issue by comradekingu** [GeeksCAT/anem-per-feina#117]
https://weblate.geekscat.org/accounts/profile/#licenses is GPLv3-only, whereas
https://github.com/GeeksCAT/anem-per-feina/blob/master/LICENSE is MIT

I think it would make sense for both to be AGPLv3+ to have the network protection.

**Comment by XaviTorello** [comment]
Makes sense for me, but I'm not sure if we can easily change the license for our `django-job-portal` fork @comradekingu

**Comment by comradekingu** [comment]
It was, and is MIT, so that is fine. Means it can't go back upstream without upstream also changing license.
It seems the "-only" part of the translation license is in error, so changing that seems fine. Going MIT there means re-licensing the work of others, so that is a pain.
Don't know how MIT software combines with GPLv3 or AGPLv3 translation licensing, but I do know running the same for both is fine.
